src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -66,6 +66,15 @@ public class PreferenceSettings : NotificationObject, IPreferences, IRenderPreci
         /// Indicates whether analytics reporting is approved or not.
         /// </summary>
         public bool IsAnalyticsReportingApproved { get; set; }
+
+        /// <summary>
+        /// Indicates whether ADP analytics reporting is approved or not.
+        /// </summary>
+        [XmlIgnore]
+        public bool IsADPAnalyticsReportingApproved { 
+            get { return Logging.Analytics.ReportingADPAnalytics; }
+            set { Logging.Analytics.ReportingADPAnalytics = value; } 
+        }
```

Comment:
Exposed the ADP option in Preferences.
Since the optin value is not specific to Dynamo...(but a global option, common for all adks products) ..I added XmlIgnore so that we do not serialize the value.